### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by andreas-venturini**
Is it possible to get the latest version of this plugin published to dockerhub?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,28 @@
 # drone-rsync
-
 Drone plugin for rsyncing files to a remote server
 
 ## Usage
-
```
Comment (tboerger) [inline README.md:6]: Those empty lines are intended.

Comment (andreas-venturini) [inline README.md:6]: Added them back and squashed both commits.

**Comment by tboerger** [comment]
LGTM

**Comment by jackspirou** [comment]
LGTM

**Pull request merged**